locale/en.json

```diff
@@ -39,6 +39,7 @@
     "copy": "Copy"
   },
   "sidebar": {
+    "nodeFilter": "Node filter",
```

Comment:
It was just a thought and for me it is not important.
I think, this could be discussed in an issue if somebody want to change it.
-> the functionality works what is the main idea of this PR so this PR is okay for me.